repose-aggregator/components/filters/regex-rbac-filter/src/main/resources/META-INF/schema/examples/regex-rbac.cfg.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<regex-rbac xmlns="http://docs.openrepose.org/repose/regex-rbac/v1.0"
+             mask-rax-roles-403="false">
+    <resources>
+        /path/to/this   GET     role1,role2,role3,role4
+        /path/to/this   PUT     role1,role2,role3
+        /path/to/this   POST    role1,role2
+        /path/to/this   DELETE  role1
+        /path/to/that   GET,PUT ALL
+        /path/to/that   ALL     role1
+        /path/{to}/wild GET     role1
```

Comment:
can we be more obvious about the fact that these are regexs?

Reply:
I have already updated that in my local copy.
Pushing now.